docker-compose.yml

```diff
@@ -21,8 +21,7 @@ services:
         working_dir: /var/www/symfony
         environment:
             NODE_ENV: development
-        command: sh -c "yarn install && yarn encore dev"
-        restart: "no"
+        command: sh -c "yarn install && yarn encore dev --watch"
```

Comment:
Hey @alcalyn, any reason why you changed this line in this PR?

Reply:
Yes, to enable css/js watching, so that when you change javascript or css, changes are taken into account immediately. I used it to develop on js

Reply:
I understood why you added --watch, but why in this PR for automating the installation with a make file

Reply:
Because the PR is "Ease install and development environment" :D